scripts/run-in-env.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
```

Comment:
Do we really need this script? I'd assume that the contributor has the environment active when committing.

Reply:
Probably not only reason I added it was because the HA core repo has it

Reply:
Yeah, I think it was added there when some contributors were running GUI tools that didn't activate the environment automatically. I don't think we need it here as we'll have a smaller contributor community. We can update later if we see a need.

Reply:
Kk I'll remove in the morning 😃